R/bind_pdf.R

```diff
@@ -0,0 +1,24 @@
+#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'. 
+#'
+bind_pdfs = function(temp_out_dir, final_out_dir, write_file_name){
+  files = list.files(temp_out_dir, full.names=T, include.dirs=F)
+  
+  pdf(paste0(final_out_dir, "/", write_file_name))
+  for (file in files){
+    grob = grid::rasterGrob(png::readPNG(file, native = FALSE), interpolate = FALSE)
+    gridExtra::grid.arrange(grob)
+  } 
+  dev.off()
+} 
+
+parser <- argparse::ArgumentParser()
```

Comment:
This chunk with the argument parser needs to not live in the library code

Reply:
Copy lines 14-24 into a new file. Put it in subdirectory `inst/` named whatever you want (`bind_pdfs.R`?)